src/scripts/main.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+  const titleN = document.createElement('h2');
+  const descriptionN = document.createElement('p');
+
+  notification.classList.add('notification', type);
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  titleN.classList.add('title');
+  titleN.textContent = title;
+
+  descriptionN.classList.add('description');
+  descriptionN.textContent = description;
+
+  notification.appendChild(titleN);
+  notification.appendChild(descriptionN);
+
+  document.body.appendChild(notification);
+
+  setTimeout(() => {
+    notification.style.visibility = 'hidden';
+  }, 2000);
```

Comment:
Instead of setting the notification's visibility to 'hidden', consider removing the notification from the DOM using `notification.remove()` after the timeout. This will prevent unnecessary elements from remaining in the document.